content/projects/infrastructure/index.adoc

```diff
@@ -75,6 +75,8 @@ we have a follow the sun policy where we try to only be on call during our "avai
 Another major aspect of the Jenkins Infrastructure project, is the maintenance of all services either provided by us or third parties.
 Here is a non-exhaustive list of services that we provide and maintain.
 
+In case you need to add some of those service to your allow-list, you can find their public IP addresses at https://reports.jenkins.io/infrastructure/latest/index.json (updated hourly)
```

Comment:
Optional phrasing change.  Approved whether or not you apply this proposal
```suggestion
In case you need to add some of those services to your allow-list, you can find their public IP addresses at https://reports.jenkins.io/infrastructure/latest/index.json (updated hourly)
```